packages/ckeditor5-list/docs/features/document-lists.md

```diff
@@ -0,0 +1,164 @@
+
+---
+menu-title: Document lists
+category: features-lists
+order: 30
+modified_at: 2022-09-20
+---
+
+{@snippet features/lists-source}
+
+
+# Document list
+
+The document list feature, provided by a separate plugin, is based on a completely different approach tha the original {@link features/lists lists feature}. Unlike regular list, which is a content block in itself, the document list plugin will let any part of the content be part of a list. Content blocks and elements – such as blockquotes, tables, paragraphs, and others – can now be put inside a list item, ensuring the continuity of numbering and retaining indentation.
+
+Just like regular lists, the document list feature supports both ordered and unordered lists.
+
+<info-box info>
+	There are currently two plugins providing lists support for CKEditor 5: the original {@link features/lists lists feature} and this new **document lists** feature.
```

Comment:
I would prefer to use "regular lists" (or simple lists could be even better)  instead of "original lists" and use "regular" later in the guide. Let's use one version across the document.

Reply:
Its "original least feature" like in "original feature" as this one is new. But I'll remove it, as it will become deprecated at one point.